docs/monitoring.md

```diff
@@ -248,6 +248,27 @@ Security options for the Spark History Server are covered more detail in the
         written to disk will be re-used in the event of a history server restart.
     </td>
   </tr>
+  <tr>
+    <td>spark.history.custom.executor.log.url</td>
+    <td>(none)</td>
+    <td>
+        Specifies custom spark executor log url for supporting external log service instead of using resource managers' app log urls.
+        Spark will support some path variables via patterns which can be vary on resource managers. 
+        Please check the document for resource manager to refer which patterns are supported.
+        <p/>
+        For now, YARN mode only supports this configuration.
```

Comment:
Similar to the one above:

```
Specifies custom spark executor log url for supporting external log service instead of using cluster
managers' application log urls in the history server. Spark will support some path variables via patterns
which can vary on cluster manager. Please check the documentation for your each cluster manager to
see which patterns are supported, if any.  This configuration has no effect on a live application, it only
affects the history server.
<p/>
For now, only YARN mode supports this configuration
```